src/navigation/HomeStackNavigator.tsx

```diff
@@ -2,6 +2,7 @@ import { createStackNavigator } from '@react-navigation/stack'
 import React from 'react'
 import { useTheme } from 'styled-components'
 
+import { Theme } from '../constants/theme'
```

Comment:
🔧 Importing this from `native`avoids the need for the typecast below:
```suggestion
import { useTheme } from 'styled-components/native'

```

Reply:
Nice, thanks!